straxen/plugins/acqmon_processing.py

```diff
@@ -0,0 +1,138 @@
+import strax
+import straxen
+import numba
+import numpy as np
+from immutabledict import immutabledict
+
+export, __all__ = strax.exporter()
+
+# More info about the acquisition monitor can be found here:
+# https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenon1t:alexelykov:acquisition_monitor
+
+@export
+@strax.takes_config(strax.Option('hit_min_amplitude_aqmon', default = 50, track = False,
+                                 help = 'Minimum hit threshold in ADC*counts above baseline'))
+
+class AcqmonHits(strax.Plugin):
+    """ Find hits in acquisition monitor data. These hits could be 
+        then used by other plugins for deadtime calculations, 
+        GPS SYNC analysis, etc.
+    """
+    __version__ ='0.0.2'
+    
+    depends_on = ('raw_records_aqmon') 
+    provides  = ('acqmon_hits')
```

Comment:
This I find somewhat confusing. I'd prefer to also call this aqmon rather than acqmon otherwise it's inconsistent with the raw_records_aqmon

Reply:
It's acutally "a**c**quisition" not "aquisition" but I can chage it to aqmon if you like it better